scripts/cargo.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-export GXI_PLUGIN_DIR=$6
+export GXI_PLUGIN_DIR=$5
 
-cargo build --manifest-path $1/Cargo.toml --target-dir $3 --release && cp $3/$5/release/$4 $2
\ No newline at end of file
+cargo build --manifest-path $1/Cargo.toml --target-dir $3 --release && cp $3/$RUST_TARGET/release/$4 $2
```

Comment:
This doesn't work if RUST_TARGET is set but we're not cross compiling since cargo puts binaries in target/release/ then. Maybe pass something to cargo.sh depending on what meson.is_cross_build() (or however that function is actually called) returns in the meson.build for conditionalising this